conda-recipe/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set package_name = "git@github.com:pcdshub/pygrafana.git" %}
```

```suggestion
{% set package_name = "pygrafana" %}
```

This is also just the package name, not the url to the repo